From 4944880ed2227f666b0761961663b0fbbcc45aa8 Mon Sep 17 00:00:00 2001
From: Jan de Groot <jan@jgc.homeip.net>
Date: Thu, 30 Jun 2011 19:54:21 +0200
Subject: [PATCH] Port to gtk3. This raises the required version for
 gnome-bluetooth to 3.0, as this is the first released
 version that uses gtk3.

---
 configure.in |    7 +++----
 src/ui.c     |    6 ------
 2 files changed, 3 insertions(+), 10 deletions(-)

diff --git a/configure.in b/configure.in
index 2338f4b..e486768 100644
--- a/configure.in
+++ b/configure.in
@@ -13,7 +13,7 @@ dnl Initialize libtool
 AM_PROG_LIBTOOL
 
 GNOKII_REQS="gnokii >= 0.6.28"
-GNOME_BLUETOOTH_REQS="gnome-bluetooth-1.0"
+GNOME_BLUETOOTH_REQS="gnome-bluetooth-1.0 >= 3.0"
 
 AC_PROG_CC
 AC_PROG_INSTALL
@@ -44,11 +44,10 @@ PKG_CHECK_MODULES(TMP_EVO, libebook-1.2,
 
 PKG_CHECK_MODULES(LIBGSM, glib-2.0 gobject-2.0 $GNOKII_REQS gthread-2.0 bluez $evo_pc_file)
 
-PKG_CHECK_MODULES(PHONEMGR, gtk+-2.0 >= 2.18 glib-2.0 >= 2.25.0
-	libcanberra-gtk gconf-2.0
+PKG_CHECK_MODULES(PHONEMGR, gtk+-3.0 >= 3.0 glib-2.0 >= 2.25.0
+	libcanberra-gtk3 gconf-2.0
 	$GNOME_BLUETOOTH_REQS $evo_pc_file
 	gmodule-2.0 dbus-glib-1 gnome-icon-theme >= 2.19.1
-	gtkspell-2.0
 )
 
 DBUSLIBDIR="`$PKG_CONFIG dbus-glib-1 --variable=libdir`"
diff --git a/src/ui.c b/src/ui.c
index 56a70ad..bc89bf4 100644
--- a/src/ui.c
+++ b/src/ui.c
@@ -22,7 +22,6 @@
 #include <glib/gi18n.h>
 #include <stdlib.h>
 #include <gtk/gtk.h>
-#include <gtkspell/gtkspell.h>
 #include <canberra-gtk.h>
 #include <time.h>
 #include <string.h>
@@ -343,11 +342,6 @@ create_send_dialog (MyApp *app, GtkDialog *parent, const char *recip)
 	buf = gtk_text_view_get_buffer (view);
 	gtk_text_buffer_set_text (buf, "", 0);
 
-        if (!gtkspell_new_attach (view, NULL, &err)) {
-        	g_warning ("Couldn't initialise spell checking: %s", err->message);
-                g_error_free (err);
-        }
-
 	entry = GTK_ENTRY (gtk_builder_get_object (ui, "recipient"));
 	if (recip)
 		gtk_entry_set_text (entry, recip);
-- 
1.7.6

